index.js

```diff
@@ -29,21 +29,45 @@ class LogStreamingPlugin {
 	}
 
 	logServerless() {
+		let logHandlerLambdaName;
+		const logHandlerFnName = _.get(this.serverless, "service.custom.logHandler.function", "loghandler");
 		_.forEach(this.serverless.service.getAllFunctions(), functionName => {
-			if (functionName === "loghandler") {
+			const functionLogicalId = this.serverless.service.provider.naming.getLambdaLogicalId(functionName);
+			if (functionName === logHandlerFnName) {
+				// Do not add the loghandler to itself.
+				logHandlerLambdaName = functionLogicalId;
 				return;
 			}
-			const functionLogicalId = `${_.upperFirst(functionName)}LambdaFunction`;
 			const functionObject = this.serverless.service.getFunction(functionName);
-
+			if (_.get(functionObject, "loghandler", true) === false) {
+				// If the loghandler property is set to false then do not create a loghandler
+				return;
+			}
 			cloudwatchLogsSubscriptionFilterTemplate.Properties.LogGroupName = `/aws/lambda/${functionObject.name}`;
 			const newResources = {
 				[`${functionLogicalId}SubscriptionFilter`]: cloudwatchLogsSubscriptionFilterTemplate
```

Comment:
Maybe there's a naming function for the log stuff too (but I'm too lazy to look it up now 😄 )